src/api/customer/customer-serializer.ts

```diff
@@ -19,6 +19,64 @@ const createCustomerSerializer: CustomerCreationRequestHandler = (req, res, next
     next();
 }
 
+const findAllCustomerSerializer: CustomersFindAllRequestHandler = (req, res, next) => {
+    const { customerToFindAll } = res.locals;
+    if(res.locals.customerToFindAll.length == 0){
+        res.locals.customerError = {
+            statusCode: 404,
+            error: 'Not Found',
+            message: 'Validation failed',
+            validation: {
+                params: {
+                    source: 'params',
+                    keys: ['uuid'],
+                    message: 'Customers Not Found!'
+                }
+            }
+        }
+    } else{
+        res.locals.customerToRespond = res.locals.customerToFindAll
+    }
+    next();
+}
+
+const findCustomerSerializer: CustomersFindRequestHandler = (req, res, next) => {
+    
+    const { customerFind } = res.locals;
+    if(customerFind != undefined){
```

Comment:
Poderia simplificar e analisar se o valor é um `falsy value`, ao invés de checar se é `undefined`:

Ou mesmo que cheque por `undefined`, procurar usar o comparator `!==`, ao invés de apenas `!=`.
Pois o primeiro checa valor e tipo, sendo que o segundo checa apenas tipo.
Porém para esse tipo de variável que estamos utilizando (objeto) não faz muita diferença, é mais questão de design pattern mesmo.

```suggestion
    if (customerFind) {
```